specs/protocol/configurability.md

```diff
@@ -68,10 +68,11 @@ These requirements are currently a draft, pending governance approval.
 | L1 Proxy Admin | Account authorized to upgrade L1 contracts. | [L1 Proxy Admin Owner](#admin-roles) | [Batch Inbox Address](#consensus-parameters), [Start block](#consensus-parameters), [Proposer address](#service-roles), [Challenger address](#service-roles), [Guardian address](#service-roles), [Challenge Period](#consensus-parameters), [Output frequency](#policy-parameters), [L2 block time](#consensus-parameters), [L1 smart contracts](#consensus-parameters) | [ProxyAdmin.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/universal/ProxyAdmin.sol) from the latest `op-contracts/vX.Y.X` release of source code in [Optimism repository](https://github.com/ethereum-optimism/optimism). | Governance-controlled, high security. |
 | L1 ProxyAdmin owner                   | Account authorized to update the L1 Proxy Admin.  | | [L1 Proxy Admin](#admin-roles) | [0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A](https://etherscan.io/address/0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A) [^of-sc-gnosis-safe-l1] | Governance-controlled, high security. |
 | L2 Proxy Admin | Account authorized to upgrade L2 contracts. | [L2 Proxy Admin Owner](#admin-roles) | [Predeploys](./predeploys.md#overview) | [ProxyAdmin.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/universal/ProxyAdmin.sol) from the latest `op-contracts/vX.Y.X` release of source code in [Optimism repository](https://github.com/ethereum-optimism/optimism). Predeploy address:  [0x4200000000000000000000000000000000000018](https://docs.optimism.io/chain/addresses#op-mainnet-l2). | Governance-controlled, high security. |
-| L2 ProxyAdmin owner                   | Account authorized to update the L2 Proxy Admin.                                                                             |                                     | [L2 Proxy Admin](#admin-roles) | Optimism Foundation Gnosis Safe e.g. [0x7871d1187A97cbbE40710aC119AA3d412944e4Fe](https://optimistic.etherscan.io/address/0x7871d1187A97cbbE40710aC119AA3d412944e4Fe) [^of-gnosis-safe-l2] | Governance-controlled, high security. |
+| L2 ProxyAdmin owner                   | Account authorized to update the L2 Proxy Admin. This is the aliased L1 ProxyAdmin owner address.                                                                            |                                     | [L2 Proxy Admin](#admin-roles) | GnosisSafe between Optimism Foundation (OF) and the Security Council (SC). Aliased Address: [0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b](https://optimistic.etherscan.io/address/0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b) [^aliased-of-sc-gnosis-safe-l1] | Governance-controlled, high security. |
```

Comment:
Unrelated to this PR, but the ProxyAdmin is not updateable so "Account authorized to update the L2 Proxy Admin." seems like a misleading description for the L1PAO and L2PAO rows. What do you think about 
```suggestion
| L2 ProxyAdmin owner                   | Account authorized to upgrade protocol contracts via calls to the `ProxyAdmin`. This is the aliased L1 ProxyAdmin owner address.                                                                            |                                     | [L2 Proxy Admin](#admin-roles) | Gnosis Safe between Optimism Foundation (OF) and the Security Council (SC). Aliased Address: [0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b](https://optimistic.etherscan.io/address/0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b) [^aliased-of-sc-gnosis-safe-l1] | Governance-controlled, high security. |
```